AppointmentManagement.java

```diff
@@ -166,24 +167,45 @@ protected void editConsultationNotes(int patientId, String patientName, String d
         }
     }
 
+    private boolean isValidTime(String time) {

```

Comment:
Follow the code format style whenever you're working with a shared codebase, i.e. 
Instead of 
isValidTime(){
}

do 
isValidTime()
{
}
i.e. braces on different lines.